app/resources/views/pages/profile/profile.blade.php

```diff
@@ -3,11 +3,19 @@
   <link href="/css/jquery/jquery-ui.css" rel="stylesheet" type="text/css"> 
   <script src="/css/jquery/external/jquery/jquery.js"></script>
   <script src="/css/jquery/jquery-ui.js"></script>
-  <script src="/js/profile.js"></script>
+  <script src ="/js/profile_photo_upload.js"> </script>
```

Comment:
It looks like you forgot to commit changes to "profile_photo_upload.js" since that file isn't in your pull request.

Also, can you test that the accordion on the profile still works with your changes?  Test by clicking the "Select All" and other checkboxes under "My Accessibility Requirements".  It looks like you stopped referencing "profile.js" which had that functionality in it so unless you copied it over to profile_photo_upload.js, it won't work.  If you can add the profile.js reference back in, that would be better than copying the accordion functionality to profile_photo_upload.js because the accordion is unrelated to photo uploading.